src/frontend.c

```diff
@@ -2910,16 +2910,28 @@ int frontend(struct program *prg, int nfile, char **fname, dictmap_callback_t di
 					}
 				} else if(1 == sscanf(str, "font-style : %s", param)) {
 					if(!strcmp(param, "italic") || !strcmp(param, "oblique")) {
-						bc->style = STYLE_ITALIC;
+						bc->style |= STYLE_ITALIC;
+					}else if(!strcmp(param, "normal")) {
+						bc->unstyle |= STYLE_ITALIC;
 					}
 				} else if(1 == sscanf(str, "font-weight : %s", param)) {
 					if(!strcmp(param, "bold")) {
-						bc->style = STYLE_BOLD;
+						bc->style |= STYLE_BOLD;
+					}else if(!strcmp(param, "normal")) {
```

Comment:
Minor formatting issue:

```suggestion
					} else if(!strcmp(param, "normal")) {
```

Here and elsewhere!